docs/deployment/plugins/k8s/index.rst

```diff
@@ -6,276 +6,7 @@ Configure Kubernetes Plugins
 .. tags:: Kubernetes, Integration, Spark, AWS, GCP, Advanced
 
 This guide provides an overview of setting up the Kubernetes Operator backend plugin in your Flyte deployment.
-
-Spin up a cluster
```

Comment:
@davidmirror-ops this section is necessary for the demo cluster. before spinning up the cluster, plugin config needs to be added to `~/.flyte/sandbox/config.yaml` file.

Reply:
we can probably modify the title of the section to let the readers know that this section is applicable when spinning up a demo cluster.

Reply:
Right, my bad. Just restored the configs, adding a tab for the sandbox config for each plugin config